### DISCUSSION TITLE
[Mictosoft Store] Unable to move DBeaver CE to other disk

### DISCUSSION THREAD

**Discussion by yiding-he** [dbeaver/dbeaver#21175] [Q&A] [answered]
#### System information: 
- OS: Windows 11
- DBeaver version: 22.1.2.202207091910
- Additional extensions: None

#### Problem:

![ApplicationFrameHost_gwGhfKEWo8](https://user-images.githubusercontent.com/900606/179648064-2b1c136b-aef5-4f14-ae6d-f115342c0114.png)
_Trying to move DBeaver CE from C: to D:_

![SystemSettings_hHGnl0A2Qe](https://user-images.githubusercontent.com/900606/179648075-63acb36b-6ab4-42c7-b2c5-d164459a4752.png)
_Failed with message 'Operation finished unsuccessfully, please try again later. Error code: 0x80073d02'_

**Comment by serge-rider** [comment, marked as answer]
This is how Microsoft Store works. You can't move applications to another location. 

You can download DBeaver from dbeaver.io and put it in any location.
